src/Integration.UnitTests/Integration.UnitTests.ruleset

```diff
@@ -1,4 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="SonarQube" ToolsVersion="14.0">
   <Include Path="..\sonarqube\sonarlint-visualstudiocsharp.ruleset" Action="Default" />
+  <Rules AnalyzerId="SonarLint.CSharp" RuleNamespace="SonarLint.CSharp">
+    <Rule Id="S2360" Action="None" />
```

Comment:
Less strict for tests, although optional arguments should be off by default IMO.